src/System.Security.Cryptography.X509Certificates/ref/System.Security.Cryptography.X509Certificates.cs

```diff
@@ -66,6 +86,16 @@ public enum StoreName
         TrustedPeople = 7,
         TrustedPublisher = 8,
     }
+    public sealed partial class SubjectAlternativeNameBuilder
+    {
+        public SubjectAlternativeNameBuilder() { }
+        public void AddDnsName(string dnsName) { }
+        public void AddEmailAddress(string emailAddress) { }
+        public void AddIpAddress(System.Net.IPAddress ipAddress) { }
+        public void AddUri(System.Uri uri) { }
+        public void AddUserPrincipalName(string upn) { }
+        public System.Security.Cryptography.X509Certificates.X509Extension Build(bool critical=false) { throw null; }
```

Comment:
[RFC 5280](https://tools.ietf.org/html/rfc5280#section-4.2.1.6):

> If the subject field
   contains an empty sequence, then the issuing CA MUST include a
   subjectAltName extension that is marked as critical.  When including
   the subjectAltName extension in a certificate that has a non-empty
   subject distinguished name, conforming CAs SHOULD mark the
   subjectAltName extension as non-critical.

Since we don't have the knowledge here, and almost no one uses empty subject names, false seems the most appropriate.